po/de.po

```diff
@@ -1534,9 +1536,8 @@ msgid "Return to previous search results (if any)"
 msgstr "zum vorherigen Suchergebnis zurückkehren (sofern vorhanden)"
 
 #: src/keymap.cpp:413
-#, fuzzy
 msgid "Go to the feed of the article"
-msgstr "zum nächsten ungelesenen Artikel gehen"
+msgstr "zum Feed des Artikels gehen"
```

Comment:
I notice that a lot of translations for the help strings start in lowercase, but not all of them. Does German has a rule regarding that, or is it just an accident?

(I'm merging this PR without waiting for you answer because it's good enough as it is. Just wanted to point out an oddity in case it's actually an error.)

Reply:
Yeah, I noticed the different casing, too. But since all the other descriptions in the help dialog start in lowercase, I decided to keep this consistent. In German, nouns are always capitalized, so the descriptions that start in uppercase have a noun as their first word. I just triple checked and noticed that two messages actually divert, damn, both start with an uppercase letter.

I'm not entirely sure if there are hard rules for this situation, but half sentences – especially in lists – often start with a lowercase letter. However, to my knowledge, it's also okay to use uppercase instead. On the weekend I will do some research on that matter and then decide how to handle this correctly. In the worst case the proper consistency fix will make it into the subsequent release. :-)